zowe/zowe-cli
# Remove the 'zowe profiles' command group

**What It Does**


This PR removes the command definitions for the `zowe profiles` command group. Thus, end-users will no longer be able to run any command from the `profiles` command group. It also removes the initialization of the profiles directory that previously occurred in the Imperative.init() function.

The majority of the changes in this PR come from re-implementing all automated tests that created a V1 profile in order to perform a desired operation.

This PR also fixes numerous system tests that failed due to changes to the Zowe error message format.

**How to Test**


1. Run a `profiles` command, and see that zowe tells you that such a command does not exist. For example:
    ```
    > zowe profiles create zosmf-profile YourZosmfProfileName --host YourHost --port 123 --user YourUser --pass "YourPassword"
    Unable to perform this operation due to the following problem.
    Command failed due to improper syntax
    Unknown group: profiles
    Did you mean: files create zos-file-system?

    Command entered: "profiles create zosmf-profile YourZosmfProfileName --host YourHost --port 123 --user YourUser --pass YourPassword"
    Use "zowe --help" to view groups, commands, and options.
    ```
2. Place some existing V1 profiles into the ZOWE_CLI_HOME\profiles directory and run a command to successfully convert those profiles to a team config.
   ```
   zowe config convert-profiles
   ```

**Review Checklist**
I certify that I have:


**Additional Comments**
